### PR TITLE
OpenAPI 3 Support

### PR DESCRIPTION
**Pull request by NelsonDaniel**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -28,15 +28,17 @@
...
+        "--use:c:/work/2018/autorest-interactive"
       ],
       "cwd": "${workspaceRoot}/Samples/openapi-v2/1c-multiple-input-swaggers"
     },
     {
       "name": ".NET Core Attach",
       "type": "coreclr",
       "request": "attach",
-      "processId": "${command:pickProcess}"
+      "processId": "16016"
```
Comment (deathly809) [inline .vscode/launch.json]: Should you hardcode the processId?

**Comment by travis-sobeck** [comment]
Any ideas one when v3 will work?  Clearly swagger/smart bear doesn't seem to be making any progress with powershell in the v3 codegen so if you can do powershell and v3 .... that would be awesome  :)

**Comment by fearthecowboy** [comment]
AutoRest v3 is in beta now, go ahead and test it:

`npm install -g autorest@beta` (requires node 10.x)

https://devblogs.microsoft.com/powershell/cmdlets-via-autorest/

https://github.com/Azure/autorest/blob/master/docs/powershell/readme.md

**Comment by travis-sobeck** [comment]
The only thing I have installed is docker, because why wouldn't you :) and I don't want to need to know how a thing works, just how to make it do what it does. 
Can I just modify line 10 in https://github.com/Azure/autorest/blob/master/Dockerfile ?
I didn't see a "beta" branch or beta tagged container at https://hub.docker.com/r/azuresdk/autorest/tags
Thanks

**Comment by fearthecowboy** [comment]
closing/ superceded by v3 branch. (no worries, commits are still there!)

**Pull request closed without merging**